### PR TITLE
8304291: [AIX] Broken build after JDK-8301998

### PR DESCRIPTION
**Pull request by adamfarley**
Hi all,

This pull request contains a backport of commit [310aa934](https://github.com/openjdk/jdk/commit/310aa9347861922af5f0311e9e93a5f49dee6adc) from the [openjdk/jdk](https://git.openjdk.org/jdk) repository.

The commit being backported was authored by Matthias Baesken on 20 Apr 2023 and was reviewed by Martin Doerr, Tyler Steele and Phil Race.

Thanks!

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8304291](https://bugs.openjdk.org/browse/JDK-8304291): [AIX] Broken build after JDK-8301998 (**Bug** - P3)

### Reviewers
 * [Martin Balao](https://openjdk.org/census#mbalao) (@martinuy - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk11u-dev.git pull/1940/head:pull/1940` \
`$ git checkout pull/1940`

Update a local copy of the PR: \
`$ git checkout pull/1940` \
`$ git pull https://git.openjdk.org/jdk11u-dev.git pull/1940/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1940`

View PR using the GUI difftool: \
`$ git pr show -t 1940`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk11u-dev/pull/1940.diff">https://git.openjdk.org/jdk11u-dev/pull/1940.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk11u-dev/pull/1940#issuecomment-1584493393)

### CODE REVIEW

**Review by martinuy — APPROVED** [review]
Hi Adam,

Thanks for contributing this backport. Looks clean to me.

If we get a jdk11u Maintainer approval for the jdk11u-critical-request 
 request (https://bugs.openjdk.org/browse/JDK-8304291), I'll sponsor this PR. As commented [here](https://github.com/openjdk/jdk11u/pull/74#issuecomment-1585296387), integration cannot occur before Maintainer approval.

Regards,
Martin.-

**Comment by RealCLanger** [comment]
Hi,

@martinuy: This is a build breakage for 11.0.20 on AIX so we need it in jdk11u. So the [other PR](https://github.com/openjdk/jdk11u/pull/74) is the correct one.

@adamfarley: Please never do `/integrate` unless there is maintainer approval via the appropriate label in JBS. And please close this PR, sorry for the additional churn.

Thanks
Christoph

**Comment by adamfarley** [comment]
Thanks for letting me know Christoph. Will close this PR now.

**Pull request closed without merging**
